app/src/main/kotlin/com/wire/android/ui/home/conversationslist/AllConversation.kt

```diff
@@ -11,54 +11,68 @@ import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.unit.dp
 import com.wire.android.R
 import com.wire.android.ui.common.UserProfileAvatar
-import com.wire.android.ui.home.conversations.all.model.Conversation
-import com.wire.android.ui.home.conversations.all.model.ConversationFolder
-import com.wire.android.ui.home.conversations.all.model.NewActivity
-import com.wire.android.ui.home.conversations.all.model.toUserInfoLabel
-import com.wire.android.ui.home.conversations.common.EventBadgeFactory
-import com.wire.android.ui.home.conversations.common.RowItem
-import com.wire.android.ui.home.conversations.common.UserLabel
-import com.wire.android.ui.home.conversations.common.folderWithElements
+import com.wire.android.ui.home.conversationslist.common.EventBadgeFactory
+import com.wire.android.ui.home.conversationslist.common.RowItem
+import com.wire.android.ui.main.conversationlist.common.UserLabel
+import com.wire.android.ui.home.conversationslist.common.folderWithElements
+import com.wire.android.ui.home.conversationslist.model.Conversation
+import com.wire.android.ui.home.conversationslist.model.ConversationFolder
+import com.wire.android.ui.home.conversationslist.model.NewActivity
+import com.wire.android.ui.home.conversationslist.model.toUserInfoLabel
+
 
 @Composable
-fun AllTab(
+fun AllConversationScreen(
     newActivities: List<NewActivity>,
-    conversations: Map<ConversationFolder, List<Conversation>>
+    conversations: Map<ConversationFolder, List<Conversation>>,
+    //TODO: This is going to be replaced with proper lambda, test purpose only
+    onConversationItemClick: () -> Unit
```

Comment:
I don't think we should provide/set the ItemClick functions externally as we can't know there which Message item was clicked. Also maybe it will not be necessary to pass any `OnItemClick` lambda as we can just specify on each row item implementation, the desired behavior when they are clicked

Reply:
I see what you mean, but I'll leave as it is for now until I'm going to actually do something with those items, would be easier to think how to refactor it by then